scss/list.scss

```diff
@@ -16,9 +16,14 @@
   height: 300px;
   margin: 0 35px 20px 35px;
   padding: 26px 0 0 28px;
+  transition: all 0.3s ease-in-out;
```

Comment:
don't use `all` value, you only apply the properties that you need

Reply:
ready 💯